src/Compilers/CSharp/Test/Syntax/Parsing/LambdaReturnTypeParsingTests.cs

```diff
@@ -5003,6 +5003,51 @@ public void Async_06()
             EOF();
         }
 
+        [Fact]
+        public void Async_TopLevelStatement()
+        {
+            string source = "async MyMethod() => null;";
+            UsingTree(source, TestOptions.Regular9);
+            verify();
+
+            UsingTree(source, TestOptions.RegularNext);
```

Comment:
Seems like this should be Regular10 (we only need to use RegularNext when we mean Regular11 which doesn't exist yet)